docs/guides/05-building/cross-platform.md

```diff
@@ -4,4 +4,76 @@ sidebar_position: 2
 
 # Cross-Platform Compilation
 
-How to use GH Action for Building: a glance at Tauri Action.
+## How to use GH Action for Building: a glance at Tauri Action
+
+### Introduction
+
+For cross-platform compilation Tauri provides Tauri Action, a GitHub Action that builds your Web application as a Tauri native binary for macOS, Linux, and Windows, it can also release the application on GitHub releases. To set up Tauri Action you must first set up a GitHub repository, you can use this action on a repo that doesn't have Tauri configured since we automatically initialize Tauri before building, and configure it to use your Web artifacts.
```

Comment:
See the above comment, *technically* not cross-compilation, but just compilation.

Reply:
Yup, I think this this is better explained now with [cf88dee](https://github.com/tauri-apps/tauri-docs/pull/611/commits/cf88deeed9aecdd12437393517fbb1c6e750dbb1)